db/migrate/20151208193058_create_orders.rb

```diff
@@ -0,0 +1,20 @@
+class CreateOrders < ActiveRecord::Migration
+  def change
+    create_table :orders do |t|
+      t.datetime :purchase_time
+      t.string :status
+      t.string :name
+      t.string :email
+      t.string :street
+      t.string :city
+      t.string :state
+      t.integer :zip
+      t.integer :cc_num
+      t.date :cc_exp
+      t.integer :sec_code
```

Comment:
Though this is a mock app, I will mention that you would never want to persist all this information in one table IRL. It would make it much too easy for a person's credit card information to be stolen and used.  If you wanted to store something to know which credit card, you could just store the last 4 of what was passed in via the form. This is why there is such big business for paypal, square etc so that small sites never have to worry about persisting, storing, or losing customer PII! :smile: